Homework3/LZW/LZW/LZW.cs

```diff
@@ -0,0 +1,150 @@
+﻿namespace LZW;
+
+using System;
+using System.IO;
+using Bor;
+
+/// <summary>
+/// A class representing the LZW algorithm
+/// </summary>
+public class LZW
+{
+    //A function for determining the number of bytes needed to store a number
+    private static int NumberOfBytes(int number)
+    {
+        if (number < 256)
+        {
+            return 1;
+        }
+        if (number < 65536)
+        {
+            return 2;
+        }
+        if (number < 16777216)
+        {
+            return 3;
+        }
+        return 4;
+    }
+
+    /// <summary>
+    /// Function for file compression
+    /// </summary>
+    /// <param name="pathToFileToCompress">The path to the file to compress</param>
+    public static void CompressFile(string pathToFileToCompress)
+    {
+        
+        string fileName = Path.GetFileNameWithoutExtension(pathToFileToCompress);
```

Comment:
```suggestion
    {
        string fileName = Path.GetFileNameWithoutExtension(pathToFileToCompress);
```
После открывающей фигурной скобки пустая строка не ставится